dev/com.ibm.ws.security.auth.data.common/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -32,3 +32,12 @@ enabled.desc=Enables the authentication data subject cache.
 
 maxSize=Maximum cache size
 maxSize.desc=Maximum number of entries supported by the authentication data subject cache.
+
+#======= KERBEROS SETTINGS =========
+krb5Principal=Kerberos principal name
+krb5Principal.desc=The name of the principal that should be used. The principal can be a simple username or a service name.
+
+krb5Keytab=Kerberos keytab file
+krb5Keytab.desc=The path of the keytab file used to obtain the principal's secret key. If unspecified, the keytab from the \
+Kerberos configuration file will be used. If it is not specified in the Kerberos configuration file then it will look for the \
+file at {user.home}/krb5.keytab.
```

Comment:
Suggested update for 

>The path of the keytab file used to obtain the principal's secret key. If unspecified, the keytab from the Kerberos configuration file will be used. If it is not specified in the Kerberos configuration file then it will look for the file at {user.home}/krb5.keytab.

to

>The path of the keytab file that is used to obtain the principal's secret key. If unspecified, the keytab from the Kerberos configuration file is used. If it is not specified in the Kerberos configuration file, then it looks for the file at {user.home}/krb5.keytab.